src/suricata.h

```diff
@@ -135,7 +135,7 @@ typedef struct SCInstance_ {
     enum RunModes aux_run_mode;
 
     char pcap_dev[128];
-    char *sig_file;
+    const char *sig_file;
```

Comment:
this is not related to app-layer, so should go into its own commit

Reply:
Actually, this cannot be made `const` as it will be used in some cases in argument for `dirname`